test/integration/parachain/staging/sequential/replace.test.ts

```diff
@@ -102,7 +119,38 @@ describe("replace", () => {
             assert.equal(requestsList[0].btcAddress, firstMapEntry.value.btcAddress);
             assert.equal(requestsList[0].amount.toString(), firstMapEntry.value.amount.toString());
             assert.equal(requestsList[0].btcHeight.toString(), firstMapEntry.value.btcHeight.toString());
-        }).timeout(400000);
+        }).timeout(1000000);
+
+        it("should fail vault replace request if not having enough tokens", async () => {
+            const replaceAmount = dustValue;
+
+            interBtcAPI.setAccount(vault_2);
+
+            // check precondition: vault does not hold enough issued tokens to request a replace
+            const tokensInVault = await interBtcAPI.vaults.getIssuedAmount(
+                newAccountId(api, vault_2.address),
+                tickerToCurrencyIdLiteral(collateralCurrency.ticker)
```

Comment:
I think you intended to use the wrapped currency here. Something like `tickerToCurrencyIdLiteral(vault_2_id.currencies.wrapped.ticker)`. Unfortunate that `.toBig(wrappedCurrency.base)` didn't catch it, I think it's because the unit value is the same.

Reply:
Ah, thank you for the catch.
I was stumbling around `vault_2_id` 's fields to find something useful. That's the one I couldn't find.

Reply:
Ah, I got confused myself because of the signature. The collateral argument is for the vault id. That's probably another sign to refactor signature to use `VaultId`s everywhere. So your initial implementation was fine, but this is indeed a bit cleaner.